modules/installation-initializing.adoc

```diff
@@ -333,7 +333,7 @@ The installation program connects to your vCenter instance.
 +
 [NOTE]
 ====
-After you create the installation configuration file, you can modify the file to create a multiple vSphere datacenters environment. This means that you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
+After you create the installation configuration file, you can modify the file to create a multiple vSphere data centers environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```

Comment:
Beyond the scope, but.....

```suggestion
After you create the installation configuration file, you can modify the file to create a multiple vSphere data centers environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```
```suggestion
After you create the installation configuration file, you can modify the file to create a multiple vSphere data center environment. This means that you can deploy an {product-title} cluster to multiple vSphere data centers that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```

Reply:
not at all! might as well fix it :nerd_face: thanks!